test/createModel.spec.js

```diff
@@ -0,0 +1,45 @@
+/* global describe, it */
+import chai, { expect } from 'chai';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+
+import BaseModel from '../src/Model';
+import createModel from '../src/createModel';
+chai.use(sinonChai);
```

Comment:
This is becoming repetitive here. Can we use `--require` on `mocha.opts` to load a helper or something? It doesn't need to be in this PR, just wanted to know your opinion and I can make the change in a separate PR along with other tests.